source/index.html.md

```diff
@@ -540,25 +554,102 @@ Exactly one payment method must be used.
   <dd>
     Amount in minor units of given currency (e.g. cents if in Euro).
   </dd>
+
   <dt>currency
     <span class="type">[A-Z]{3}</span>
   </dt>
   <dd>
     <a target="_blank" href="currencies.txt">3-letter currency code</a>. (Some exponents differ from ISO 4217.)
   </dd>
+
+  <dt>credential
+    <span class="type">(<code>storing</code>|<code>using</code>)</span>
```

Comment:
```suggestion
    <span class="type">(<code>storing</code>|<code>from_storage</code>)</span>
```
WDYT @clearhaus/pci?

Reply:
`from_storage` might be a tad better than `using` because we have chosen `credential` rather than `cof`.

Reply:
Let's "unfold" the `cof` stuff over there: https://github.com/clearhaus/gateway-api-docs/pull/111/files#r497447364

Reply:
Migrated to https://github.com/clearhaus/gateway-api-docs/pull/111/files#r524296900.